govcd/catalog_test.go

```diff
@@ -5,7 +5,19 @@
 package govcd
 
 import (
+	"github.com/vmware/go-vcloud-director/util"
 	. "gopkg.in/check.v1"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"runtime"
+	"strings"
+)
+
+const (
+	OvaPath        = "/test_vapp_template.ova"
+	OvaChunkedPath = "/template_with_custom_chunk_size.ova"
```

Comment:
1. I would agree with https://medium.com/@matryer/5-simple-tips-and-tricks-for-writing-unit-tests-in-golang-619653f90742 and move all test to different package - in that away we can avoid more clutter in future and separate stuff. And solve issue mentioned by you as binary files would be in different package. Otherwise we create issue that test exist but they fail for others if they won't checkout special folder with binary files. 

2. Yes, I can move constants to api_vcd_test.go

Reply:
We have already tests that depend on user defined artifacts. This test should be no different. It may pass with the artifacts that we recommend, but we must see how it behaves with user defined items.

Reply:
Move binaries to test-resources. Ova paths are configured if empty, tests are skipped.